src/util/isDate.ts

```diff
@@ -1,3 +1,3 @@
-export function isDate(value) {
+export function isDate(value: any): value is Date {
```

Comment:
return type `value is Date` is actually neat, I didn't aware of this. Thanks for bringing this :)